wiki/Ranking_criteria/en.md

```diff
@@ -92,16 +92,19 @@ Important to understand before using:
 - **A [beatmap host](/wiki/Beatmap/Beatmap_host) must have beatmapped equal or more difficulties than any [guest difficulty](/wiki/Beatmap/Guest_difficulty) creator.** This is to provide credit where credit is due. [Collaborative difficulties](/wiki/Beatmap/Beatmap_collaborations) are only considered partial difficulties, and [drain time](/wiki/Beatmap/Drain_time) will be used to determine the amount of contribution when a guest has beatmapped significantly more than the host.
 - **A [beatmap host](/wiki/Beatmap/Beatmap_host) and [guest difficulty](/wiki/Beatmap/Guest_difficulty) creators can make changes to their respective difficulties as they wish.** If there is a disagreement between both, the beatmap host must delete the guest contribution upon request. If a guest difficulty creator cannot be contacted for a month, they will be assumed to agree with any changes.
 - **Beatmaps cannot contain significant portions of difficulties belonging to other users without their explicit involvement or permission.**
 
 ### Guidelines
 
 - **The highest difficulty of a beatmap should correspond to the general feel of the song.** Easy/Normal difficulties can be used as a lone difficulty of a beatmap if their rhythms are not oversimplified. A Hard difficulty or beyond should be included otherwise.
-- **Avoid incomprehensible username combinations to indicate possession of a [collaborative difficulty](/wiki/Beatmap/Beatmap_collaborations).** If it's unclear whose usernames are combined, simplification is recommended.
-- **Avoid difficulty names with descriptive elements not clearly related to a [guest difficulty](/wiki/Beatmap/Guest_difficulty) creator or a level of difficulty.** (e.g. Beatmap Creator's Tragic Love Extra)
+- **Difficulty names should not be misleading.**
+  - **Descriptive difficulty names should have a relation to the song.** When adding other terms such as `Tragic Love` to a difficulty name, it should relate to the song.
+  - **When combining multiple usernames into a single collaborator name for a [guest difficulty](/wiki/Beatmap/Guest_difficulty), make sure the name remains comprehensible.** For example `Mamayucha` being a [collaboration difficulty](/wiki/Beatmap/Beatmap_collaborations) between `Mani`, `Mari`, `Yuzu` and `Mocha` loses many of the collaborators' features and listing their full usernames, replacing the collaborative username with `Collab` or a descriptor like `osu! Mascots' Expert` may be clearer.
+  - **The host should not include difficulties such as `Host's Insane`, unless this is done in order to rank the same metadata twice or strongly encouraged by a naming scheme related to the song.**
+  
+  If it is unclear, a discussion should be held to come to a consensus on whether the difficulty name can be used or should be changed.
```

Comment:
```suggestion
  - **The host should not include difficulties such as `Host's Insane`, unless this is done in order to rank the same metadata twice or strongly encouraged by a naming scheme related to the song.** If it is unclear, a discussion should be held to come to a consensus on whether the difficulty name can be used or should be changed.
```

that or

`- **Difficulty names should not be misleading.** If it is unclear, a discussion should be held to come to a consensus on whether the difficulty name can be used or should be changed.`

not sure which one was meant here

Reply:
both suggestions are wrong, they are supposed to visually be on the same level as `Difficulty names should not be misleading` as you've guessed

Reply:
to specify: the sentence was at the end and under the list because if none of the bulletpoints apply to your usecase and you're not sure, have a discussion before ranking the map was the idea, moving it up kinda makes it read worse to me

Reply:
what does "rank the same metadata twice" mean? it's very weird terminology.

Reply:
@peppy https://github.com/ppy/osu-stable-issues/issues/714 basically this usecase is when people are generally fine with including themselves as a difficulty owner in the format `Host's <Difficulty Level>` 

i.e. 
- https://osu.ppy.sh/beatmapsets/474731#osu/1014240 Ranked in July 2016
- https://osu.ppy.sh/beatmapsets/488619#osu/1045500 Ranked in September 2016 had to use `Doormat's Easy` to avoid osu! thinking the difficulty is from the ranked set 

I didn't know a better way to word this while keeping it brief. Maybe I should elaborate in a footnote?

Reply:
the point is that it messes up list spacing ([image](https://github.com/user-attachments/assets/e2e450e3-9fe5-449e-84d5-6ed9851d3e11)), which is probably why lists aren't formatted this way elsewhere on the wiki. if it's to be on the same level then it should be just another bullet.

```suggestion
  - **The host should not include difficulties such as `Host's Insane`, unless this is done in order to rank the same metadata twice or strongly encouraged by a naming scheme related to the song.**
  - If it is unclear, a discussion should be held to come to a consensus on whether the difficulty name can be used or should be changed.
```